app/jobs/chat_channels/index_job.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module ChatChannels
+  class IndexJob < ApplicationJob
+    queue_as :chat_channels_index
+
+    def perform(chat_channel_id:)
+      chat_channel = ChatChannel.find_by(id: chat_channel_id)
+
+      return unless chat_channel
```

Comment:
@lightalloy what do you think about raising an exception if the record is not found?

Reply:
In this case, there is not much value in raising exceptions. It would mean only that the ChatChannel was destroyed and we don't need to reindex it. In some cases, it's worth raising exceptions in the jobs to see that something went wrong and we need to take action but it's not the case. As @rhymes said, the jobs will be retried and waste resources which we don't want.